src/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -9,6 +9,7 @@
     <FileAlignment>512</FileAlignment>
     <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
     <GenerateAppxPackageOnBuild>False</GenerateAppxPackageOnBuild>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard1.0' or '$(TargetGroup)' == 'netstandard1.3'">netstandard1.0;portable-net45+win8+wp8+wpa81</PackageTargetFramework>
```

Comment:
@weshaggard should this be a separate condition so that when it targets ns1.3 we actually include ns1.3 for the packagetargetframework? Or we’re fine with only including ns1.0 as the TFM?

Reply:
Actually I made the incorrect assumption that this was the same as before. It should just be `<PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard1.0'">netstandard1.0;portable-net45+win8+wp8+wpa81</PackageTargetFramework>` like before. This is saying put the netstandard1.0 asset in the portable... folder as well we don't want multiple configurations putting into the same portable folder in the package.